selfdrive/ui/paint.cc

```diff
@@ -404,6 +404,21 @@ static void ui_draw_world(UIState *s) {
     draw_chevron(s, scene->lead_d_rel+2.7, scene->lead_y_rel, 25,
                   nvgRGBA(201, 34, 49, fillAlpha), nvgRGBA(218, 202, 37, 255));
   }
+  if (scene->lead_status2) {
+    // Draw lead2 car indicator
```

Comment:
Can you refactor this duplicated code into a `draw_lead(scene, d_rel, v_rel, y_rel)` function?

Reply:
let us see if the tests come back with a tick